clap_man/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "clap_man"
+version = "3.0.0-rc.4"
+edition = "2018"
+include = [
+	"src/**/*",
+	"Cargo.toml",
+	"LICENSE-*",
+	"README.md"
+]
+description = "A generator library used with clap for manpages"
```

Comment:
I'd simplify his to "A manpage generator for clap"

Reply:
Done.